src/main/java/com/google/api/generator/gapic/composer/MethodSampleCodeHelperComposer.java

```diff
@@ -82,6 +87,49 @@ public static TryCatchStatement composeUnaryRpcMethodSampleCode(
         .build();
   }
 
+  public static TryCatchStatement composeUnaryPagedRpcMethodSampleCode(
+      Method method,
+      List<MethodArgument> arguments,
+      TypeNode clientType,
+      Map<String, ResourceName> resourceNames,
+      TypeNode pagedItemType) {
+    VariableExpr clientVarExpr = createVariableExpr(getClientName(clientType), clientType);
+    // Assign each method arguments with its default value.
+    Map<String, VariableExpr> methodArgVarExprMap = createMethodArgumentsVariableExprs(arguments);
+    List<Expr> methodArgumentsAssignmentExpr =
+        assignMethodArgumentsWithDefaultValues(arguments, methodArgVarExprMap, resourceNames);
+    List<Expr> methodArgVarExprs =
+        arguments.stream()
+            .map(arg -> methodArgVarExprMap.get(arg.name()))
+            .collect(Collectors.toList());
+    // For loop client on iterateAll method.
+    // e.g. for (LogEntry element : loggingServiceV2Client.ListLogs(parent).iterateAll()) {
+    //          //doThingsWith(element);
+    //      }
+    VariableExpr elementVarExpr = createVariableExpr("element", pagedItemType);
```

Comment:
Same as in #573, let's use `VariableExpr.builder().setName("element").setType(pagedItemType)` directly instead of the wrapper. In this case, since it's used only in the ForStatement, we can even inline it into the `setLocalVariableExpr` method.

Reply:
SG